src/api/resources/todos_resource.py

```diff
@@ -0,0 +1,41 @@
+from flask import jsonify, abort
+from flask_restful import Resource, reqparse
+
+from src.data_access.builders.todo_builder import TodoBuilder
+from src.middleware.auth.auth_context import get_auth_context
+from src.middleware.exc.todos import *
+from src.middleware.todo_store import TodoStore
+
+parser = reqparse.RequestParser()
+
+parser.add_argument('datetime')
+parser.add_argument('title')
+
+
+class TodosResource(Resource):
+    def __init__(self, todo_store: TodoStore):
+        self._todo_store = todo_store
+
+    def post(self):
+        args = parser.parse_args()
+        try:
+            title, datetime = args['title'], args['datetime']
+            user_id = get_auth_context()
+            todo = TodoBuilder(user_id, title) \
```

Comment:
I dont know if I would put it in another class, maybe I would put it in one function (that can be splitted to private methods) cause set_date_time and build always will be called together